smbutil.c

```diff
@@ -800,10 +801,11 @@ smb_fdata1(netdissect_options *ndo,
 	    }
 	    if (t != 0) {
 		lt = localtime(&t);
-		if (lt != NULL)
-		    tstring = asctime(lt);
-		else
-		    tstring = "(Can't convert time)\n";
+		if (lt != NULL) {
+	        strftime(buffer, 26, "%a %b %e %T %Y%n", lt);;
+	        tstring = buffer;
+	    } else
+		tstring = "(Can't convert time)\n";
```

Comment:
Did you mean to change this line?

Reply:
TBH not understand what you mean, sorry

Reply:
> > Did you mean to change this line?
> TBH not understand what you mean, sorry

The code before your change is:

 		lt = localtime(&t);
 		if (lt != NULL)
 		    tstring = asctime(lt);
 		else
 		    tstring = "(Can't convert time)\n";

The code after your change is:

 		lt = localtime(&t);
 		if (lt != NULL) {
 	        strftime(buffer, sizeof(buffer), "%a %b %e %T %Y%n", lt);
 	        tstring = buffer;
 	    } else
 		tstring = "(Can't convert time)\n";

The code using `strftime()` *should* be:

 		lt = localtime(&t);
 		if (lt != NULL) {
 	            strftime(buffer, sizeof(buffer), "%a %b %e %T %Y%n", lt);
 	            tstring = buffer;
 		} else
 		    tstring = "(Can't convert time)\n";

which is correctly indented, and doesn't change the lines after the `asctime()` call.

Reply:
Understand now thanks!